package/MDAnalysis/coordinates/XTC.py

```diff
@@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
     they require significantly less disk space and the loss of precision is
     usually not a problem.
 
-    Parameter
-    ---------
-    filename : str
+    Parameters
+    ----------
+    filename: str
```

Comment:
I'll have a look. But most likely not before Monday. 
Is it a good time to mention that I do not like that space before the column? It looks inconsistent with the rest of the language (dict,  blocks). 
Le 22 juil. 2016 6:32 PM, Max Linke <notifications@github.com> a écrit :In package/MDAnalysis/coordinates/XTC.py:
> @@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
>      they require significantly less disk space and the loss of precision is
>      usually not a problem.
>  
> -    Parameter
> -    ---------
> -    filename : str
> +    Parameters
> +    ----------
> +    filename: str

the space is also there in napoleons own docs.

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

maybe you can update your version or write a bug report to them.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.